common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -256,7 +260,9 @@ public void run() {
                         // commandResult = sCommandResultCache.get(command);
                         //If nothing in cache, execute the command and cache the result
 //                        if (commandResult == null) {
-                            commandResult = executeCommand(command);
+                        CodeMarkerManager.getInstance().markCode(PerfConstants.CodeMarkerConstants.ACQUIRE_TOKEN_SILENT_COMMAND_EXECUTION_START);
+                        commandResult = executeCommand(command);
+                        CodeMarkerManager.getInstance().markCode(PerfConstants.CodeMarkerConstants.ACQUIRE_TOKEN_SILENT_COMMAND_EXECUTION_END);
```

Comment:
🐛 What happens if the command throws an exception? You'll never hit this perf marker